rust_src/crates/webrender/src/wrterm.rs

```diff
@@ -423,6 +424,15 @@ pub fn x_open_connection(
     _xrm_string: LispObject,
     _must_succeed: LispObject,
 ) -> LispObject {
+    let display = if unsafe { NILP(display) } {
```

Comment:
`display.is_nil()`

Reply:
In remacs we've used Option<LispStringRef> as argument in those situations. As far as I remember the function should throw an error if display is not a string or nil.

https://github.com/remacs/remacs/blob/master/rust_src/src/windows.rs#L1196

Reply:
Thanks